cli/src/pcluster/config/cluster_config.py

```diff
@@ -2651,6 +2651,7 @@ def _register_validators(self, context: ValidatorContext = None):
                     instance_type=instance_type,
                     capacity_reservation_id=cr_target.capacity_reservation_id if cr_target else None,
                 )
+                self._register_validator(FeatureRegionValidator, feature=self.capacity_type)
```

Comment:
Here is a bug. The fix should be something like:
`self._register_validator(FeatureRegionValidator, feature=self.capacity_type, region=self.region)`